packages/office-addin-dev-certs/src/uninstall.ts

```diff
@@ -32,13 +32,24 @@ export function deleteCertificateFiles(certificateDirectory: string = defaults.c
    }
 }
 
-export async function uninstallCaCertificate(machine: boolean = false, verbose: boolean = true) {
-   if (isCaCertificateInstalled()) {
+export async function uninstallCaCertificate(machine: boolean = false, verbose: boolean = true, expiredCert = false) {
```

Comment:
Let's not have this expiredCert parameter. isCaCertificateInstalled() should be true for an installed cert whether expired or not. There can be another isCaCertificateExpired() which checks expiration. The logic can be refactored to use those.